keps/sig-scheduling/785-scheduler-component-config-api/README.md

```diff
@@ -415,3 +414,4 @@ N/A
 - 2021-08-06: Introducing `v1beta3`.
 - 2021-09-01: Remove `v1beta1` and the legacy policy config API.
 - 2022-05-18: Introducing `v1` and mark `v1beta2` as deprecated.
+- 2022-09-05: Remove `v1beta2` in 1.26.
```

Comment:
Also add this in `Proposal`

Reply:
Thanks, it makes sense. I have updated the `Proposal` part.